spinoso-time/src/time/tzrs/to_a.rs

```diff
@@ -0,0 +1,78 @@
+use super::Time;
+
+/// Serialized representation of a timestamp using a ten-element array of
+/// datetime components.
+///
+/// [sec, min, hour, day, month, year, wday, yday, isdst, zone]
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ToA {
+    /// The second of the minute `0..=59` for the source _time_.
+    pub sec: u8,
+    /// The minute of the hour `0..=59` for the source _time_.
+    pub min: u8,
+    /// The hour of the day `0..=23` for the source _time_.
+    pub hour: u8,
+    /// The day of the month `1..=n` for the source _time_.
+    pub day: u8,
+    /// The month of the year `1..=12` for the source _time_.
+    pub month: u8,
+    /// The year (including the century) for the source _time_.
+    pub year: i32,
+    /// An integer representing the day of the week, `0..=6`, with Sunday == 0
+    /// for the source _time_.
+    pub wday: u8,
+    /// An integer representing the day of the year, `1..=366` for the source
+    /// _time_.
+    pub yday: u16,
+    /// Whether the source _time_ occurs during Daylight Saving Time in its time
+    /// zone.
+    pub isdst: bool,
+    /// The timezone used for the source _time_.
+    pub zone: String,
+}
+
+impl ToA {
+    /// `ToA` represents ten-element array of values for time:
+    ///
+    /// [sec, min, hour, day, month, year, wday, yday, isdst, zone]
+    pub const ELEMENTS: usize = 10;
+
+    /// A ten-element array of values for time:
+    ///
+    /// [sec, min, hour, day, month, year, wday, yday, isdst, zone]
+    #[inline]
+    #[must_use]
+    pub fn to_tuple(&self) -> (u8, u8, u8, u8, u8, i32, u8, u16, bool, String) {
```

Comment:
This function isn't actually used anywhere from what I can see, I just copied from the chrono impl

Reply:
It should be used here, but not currently implemented:

https://github.com/artichoke/artichoke/blob/06e9e0d88dc7d7451dc5b2269e9d05c69bef5da6/artichoke-backend/src/extn/core/time/trampoline.rs#L174-L179

Reply:
In which case, is there an easy tuple => array somewhere? I'd image this trampoline method would create an array from `crate::core::extn::array::Array`, but I'm not aware of any tuple => array magic that would make that easier than just assigning the values in order via the `pub` accessors

Reply:
yes that makes sense.

we can nuke this crazy tuple method.

Reply:
Nuked!